nav2_bt_navigator/behavior_trees/follow_point.xml

```diff
@@ -0,0 +1,20 @@
+<!--
+  This Behavior Tree replans the global path after every 1m.
+-->
+
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <PipelineSequence name="NavigateWithReplanning">
+      <RateController hz="1.0">
+        <UpdateGoal input_goal="{goal}" output_goal="{mod_goal}">
+          <ComputePathToPose goal="{mod_goal}" path="{path}" planner_id="GridBased"/>
```

Comment:
mod_path variable needs to be better named to describe what you're accomplishing

Reply:
Ok. I'll change by `updated_goal`. I will also use `truncated_path`

Reply:
I like those